railties/lib/rails/generators/rails/app/templates/config/initializers/new_framework_defaults_6_0.rb.tt

```diff
@@ -18,3 +18,6 @@
 
 # Return false instead of self when #enqueue method was aborted from the callback
 Rails.application.config.active_job.return_false_on_aborted_enqueue = true
+
+# Use ActionMailer::MailDeliveryJob for sending parameterized and normal mail.
+Rails.application.config.action_mailer.delivery_job = ActionMailer::MailDeliveryJob
```

Comment:
I think this should be commented and I'd explain why you should not deploy with this just after the upgrade.

Reply:
Also, should not the Parameterized job keep enqueing with the old job when this config is false? Otherwise the Rails 5.2 app will not be able to run the new job if it a old worker gets it.

Reply:
This message seems needlessly shouty. Perhaps we could borrow the wording from the cookies with metadata setting:

```
 # Embed purpose and expiry metadata inside signed and encrypted
 # cookies for increased security.
 #
 # This option is not backwards compatible with earlier Rails versions.
 # It's best enabled when your entire app is migrated and stable on 6.0.
```

Or perhaps we should add a section for "toggle once upgrade is stable"/"backwardsincompatible"-configs.

Reply:
I've combined the wording and tried to make it sound less like I'm yelling 😛